hazelcast/src/main/java/com/hazelcast/client/impl/spi/impl/SmartClientInvocationService.java

```diff
@@ -88,9 +88,11 @@ public void handleBackupEvent(long sourceInvocationCorrelationId) {
     public void invokeOnPartitionOwner(ClientInvocation invocation, int partitionId) throws IOException {
         final Address owner = partitionService.getPartitionOwner(partitionId);
         if (owner == null) {
+            partitionService.firstArrangement();
```

Comment:
`getPartitionOwner` already calls `firstArrangement()`, do we need it here as well?